index.bs

```diff
@@ -644,8 +651,52 @@ authorizing an authenticator with which to complete the operation.
         the credential, or that additional information be returned in the attestation statement. Alternatively, the caller may
         specify an additional message that they would like the authenticator to display to the user. Extensions are defined in
         [[#extensions]].
+
+    - The <dfn>attachment</dfn> parameter contains authenticator attachment descriptions, which is used as an additional
+        constraint on which authenticators are eligible to participate in a [[#makeCredential]] or [[#getAssertion]] operation.
+        See [[#attachment]] for a description of the attachment values and their meanings.
+
 </div>
 
+### Credential Attachment enumeration (enum <dfn enum>Attachment</dfn>) ### {#attachment}
+
+<pre class="idl">
+    enum Attachment {
+        "platform",
+        "cross_platform"
+    };
+</pre>
+
+<div dfn-for="Attachment">
+    Clients may communicate with authenticators using a variety of mechanisms.
+    For example, a client may use a platform-specific API to communicate
+    with an authenticator which is physically bound to a platform.  On the
+    other hand, a client may use a variety of cross-platform transport
+    protocols such as Bluetooth (see [[#transport]] to discover and communicate
+    with authenticators.  We define authenticators that are part of the
+    client's platform as having a |platform| attachment, while those that are
+    reachable via standardized cross-platform transport protocols are defined
+    as having |cross_platform| attachment.  This distinction is important
```

Comment:
I'll leave it to others to bikeshed about `roaming` vs other forms, but I think having such a term is more natural to my mind than "cross-platform transport", which kind of implies changing OS rather than changing device.